mkt/webapps/tasks.py

```diff
@@ -371,13 +358,6 @@ def dump_app(id, **kw):
     return target_file
 
 
-@task
-def clean_apps(pks, **kw):
```

Comment:
`clean_apps()` was called by `process_addons --task=dump_apps` which we have not used in a while. `export_data` command does this directly.